src/main/java/xyz/vegaone/easytracking/mapper/BugMapper.java

```diff
@@ -4,9 +4,13 @@
 import xyz.vegaone.easytracking.domain.BugEntity;
 import xyz.vegaone.easytracking.dto.Bug;
 
+import java.util.List;
+
 @Mapper(componentModel = "Spring")
 public interface BugMapper {
 
```

Comment:
@Mapper(componentModel = "Spring")
  public interface BugMapper {

You added MapStruct. Nice touch!